docs/install/vendor-options.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Commercial Knative Offerings"
+linkTitle: "Knative Vendors"
+weight: 15
+type: "docs"
+---
+
+Knative has a rich community with many vendors participating, and many of those
+vendors offer commercial Knative offerings.  Please check with each of these
+vendors for what is or is not supported.
+
+<!-- TODO: Talk about criteria for inclusion here?  a la Conformance -->
+
+Here is a list of commercial Knative products (alphabetically):
+
+- [Google Cloud Run for Anthos](https://cloud.google.com/run/docs/gke/setup): <!-- TODO(ryangregg): add a description -->
```

Comment:
@rgregg I'd like to include a short description of each of these (also please feel free to suggest a different name/link as appropriate.

Reply:
Added.